cypress/integration/RichTextEditor.spec.ts

```diff
@@ -17,16 +19,18 @@ describe('Rich Text Editor', () => {
 
   beforeEach(() => {
     cy.visit('/rich-text');
-    const wrapper = cy.findByTestId('rich-text-editor-integration-test').should('be.visible');
-    editor = wrapper.find('[data-slate-editor=true]').should('be.visible');
+    const wrapper = () => cy.findByTestId('rich-text-editor-integration-test');
+    editor = () => wrapper().find('[data-slate-editor=true]');
```

Comment:
I was getting some problems because I needed to use some `editor` assertions multiple times in the same test. The first time succeeded, but others failed.